cli/tests/pcluster/configure/test_pcluster_configure/test_with_input_no_automation_no_errors_with_config_file/pcluster.config.ini

```diff
@@ -7,10 +7,12 @@ base_os = ubuntu1604
 vpc_settings = default
 scheduler = slurm
 master_instance_type = c5.xlarge
-compute_instance_type = g3.8xlarge
-max_queue_size = 18
-initial_queue_size = 7
+compute_instance_type = t2.micro
```

Comment:
is it correct to have compute_instance_type when having queue_settings?

Reply:
Good catch! 
There was a problem in the HitConverter which was not cleaning up the config_parser from the old cluster parameters. Fixed now.